classes/local/store/object_file_system.php

```diff
@@ -995,4 +995,40 @@ protected function get_trash_fullpath_from_hash($contenthash) {
         debugging('Objectfs does not implement a trashdir.');
         return '';
     }
+
+    /**
+     * Add the supplied file to the file system and update its location.
```

Comment:
```suggestion
     * Add the supplied file to the file system and update its object location.
```
To me, location is too generic of a word. I've provided a suggestion but happy for any alternatives that might make it clearer.